src/L1/rollup/L1MessageQueue.sol

```diff
@@ -64,8 +64,11 @@ contract L1MessageQueue is OwnableUpgradeable, IL1MessageQueue {
     /// @dev The bitmap for skipped messages, where `skippedMessageBitmap[i]` keeps the bits from `[i*256, (i+1)*256)`.
     mapping(uint256 => uint256) private skippedMessageBitmap;
 
+    /// @inheritdoc IL1MessageQueue
+    uint256 public finalizedQueueIndexPlusOne;
```

Comment:
Can rename to `totalFinalizedL1Messages`

Reply:
rename to `nexrUnfinalizedQueueIndex`